### PR TITLE
[#8] Add query haskell repos

### PR DESCRIPTION
**Pull request by willbasky**
Resolves [#8](https://github.com/kowainik/issue-wanted/issues/8)

### CODE REVIEW

**Comment by ChShersh** [comment]
@willbasky Could you, please, post here response with couple projects? Just to check how it works.

**Comment by willbasky** [comment]
```json
*Search> fetchAllHaskellRepos
Right (SearchResult {searchResultTotalCount = 79926, searchResultResults = [Repo {repoSshUrl = Just (URL "git@github.com:jgm/pandoc.git"), repoDescription = Just "Universal markup converter", repoCreatedAt = Just 2010-03-20 20:34:23 UTC, repoHtmlUrl = URL "https://github.com/jgm/pandoc", repoSvnUrl = Just (URL "https://github.com/jgm/pandoc"), repoForks = Just 1499, repoHomepage = Just "http://johnmacfarlane.net/pandoc", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/jgm/pandoc.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/jgm/pandoc.git"), repoSize = Just 33650, repoUpdatedAt = Just 2018-09-18 11:39:35 UTC, repoWatchers = Just 12996, repoOwner = SimpleOwner {simpleOwnerId = Id 3044, simpleOwnerLogin = N "jgm", simpleOwnerUrl = URL "https://api.github.com/users/jgm", simpleOwnerAvatarUrl = URL "https://avatars0.githubusercontent.com/u/3044?v=4", simpleOwnerType = OwnerUser}, repoName = N "pandoc", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-18 11:33:48 UTC, repoId = Id 571770, repoUrl = URL "https://api.github.com/repos/jgm/pandoc", repoOpenIssues = Just 511, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/jgm/pandoc/hooks", repoStargazersCount = 12996},Repo {repoSshUrl = Just (URL "git@github.com:koalaman/shellcheck.git"), repoDescription = Just "ShellCheck, a static analysis tool for shell scripts", repoCreatedAt = Just 2012-11-17 03:15:11 UTC, repoHtmlUrl = URL "https://github.com/koalaman/shellcheck", repoSvnUrl = Just (URL "https://github.com/koalaman/shellcheck"), repoForks = Just 623, repoHomepage = Just "https://www.shellcheck.net", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/koalaman/shellcheck.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/koalaman/shellcheck.git"), repoSize = Just 3361, repoUpdatedAt = Just 2018-09-18 11:31:34 UTC, repoWatchers = Just 12287, repoOwner = SimpleOwner {simpleOwnerId = Id 2818758, simpleOwnerLogin = N "koalaman", simpleOwnerUrl = URL "https://api.github.com/users/koalaman", simpleOwnerAvatarUrl = URL "https://avatars0.githubusercontent.com/u/2818758?v=4", simpleOwnerType = OwnerUser}, repoName = N "shellcheck", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-18 01:07:48 UTC, repoId = Id 6731432, repoUrl = URL "https://api.github.com/repos/koalaman/shellcheck", repoOpenIssues = Just 355, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/koalaman/shellcheck/hooks", repoStargazersCount = 12287}
```
```
*Search> fetchHaskellReposHW
Right (SearchResult {searchResultTotalCount = 301, searchResultResults = [Repo {repoSshUrl = Just (URL "git@github.com:commercialhaskell/stack.git"), repoDescription = Just "The Haskell Tool Stack", repoCreatedAt = Just 2015-04-29 20:56:42 UTC, repoHtmlUrl = URL "https://github.com/commercialhaskell/stack", repoSvnUrl = Just (URL "https://github.com/commercialhaskell/stack"), repoForks = Just 629, repoHomepage = Just "http://haskellstack.org", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/commercialhaskell/stack.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/commercialhaskell/stack.git"), repoSize = Just 14466, repoUpdatedAt = Just 2018-09-18 00:37:00 UTC, repoWatchers = Just 2848, repoOwner = SimpleOwner {simpleOwnerId = Id 10633049, simpleOwnerLogin = N "commercialhaskell", simpleOwnerUrl = URL "https://api.github.com/users/commercialhaskell", simpleOwnerAvatarUrl = URL "https://avatars1.githubusercontent.com/u/10633049?v=4", simpleOwnerType = OwnerOrganization}, repoName = N "stack", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-17 00:55:32 UTC, repoId = Id 34817922, repoUrl = URL "https://api.github.com/repos/commercialhaskell/stack", repoOpenIssues = Just 785, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/commercialhaskell/stack/hooks", repoStargazersCount = 2848},Repo {repoSshUrl = Just (URL "git@github.com:carp-lang/Carp.git"), repoDescription = Just "A statically typed lisp, without a GC, for real-time applications.", repoCreatedAt = Just 2016-01-11 12:54:09 UTC, repoHtmlUrl = URL "https://github.com/carp-lang/Carp", repoSvnUrl = Just (URL "https://github.com/carp-lang/Carp"), repoForks = Just 81, repoHomepage = Just "", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/carp-lang/Carp.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/carp-lang/Carp.git"), repoSize = Just 7010, repoUpdatedAt = Just 2018-09-18 06:06:05 UTC, repoWatchers = Just 2321, repoOwner = SimpleOwner {simpleOwnerId = Id 20434801, simpleOwnerLogin = N "carp-lang", simpleOwnerUrl = URL "https://api.github.com/users/carp-lang", simpleOwnerAvatarUrl = URL "https://avatars1.githubusercontent.com/u/20434801?v=4", simpleOwnerType = OwnerOrganization}, repoName = N "Carp", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-15 08:05:24 UTC, repoId = Id 49426320, repoUrl = URL "https://api.github.com/repos/carp-lang/Carp", repoOpenIssues = Just 15, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/carp-lang/Carp/hooks", repoStargazersCount = 2321}
```
```
*Search> fetchHaskellReposGFI 
Right (SearchResult {searchResultTotalCount = 97, searchResultResults = [Repo {repoSshUrl = Just (URL "git@github.com:jgm/pandoc.git"), repoDescription = Just "Universal markup converter", repoCreatedAt = Just 2010-03-20 20:34:23 UTC, repoHtmlUrl = URL "https://github.com/jgm/pandoc", repoSvnUrl = Just (URL "https://github.com/jgm/pandoc"), repoForks = Just 1499, repoHomepage = Just "http://johnmacfarlane.net/pandoc", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/jgm/pandoc.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/jgm/pandoc.git"), repoSize = Just 33650, repoUpdatedAt = Just 2018-09-18 11:39:35 UTC, repoWatchers = Just 12996, repoOwner = SimpleOwner {simpleOwnerId = Id 3044, simpleOwnerLogin = N "jgm", simpleOwnerUrl = URL "https://api.github.com/users/jgm", simpleOwnerAvatarUrl = URL "https://avatars0.githubusercontent.com/u/3044?v=4", simpleOwnerType = OwnerUser}, repoName = N "pandoc", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-18 11:33:48 UTC, repoId = Id 571770, repoUrl = URL "https://api.github.com/repos/jgm/pandoc", repoOpenIssues = Just 511, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/jgm/pandoc/hooks", repoStargazersCount = 12996},Repo {repoSshUrl = Just (URL "git@github.com:lamdu/lamdu.git"), repoDescription = Just "lamdu - towards the next generation IDE", repoCreatedAt = Just 2011-10-20 21:13:43 UTC, repoHtmlUrl = URL "https://github.com/lamdu/lamdu", repoSvnUrl = Just (URL "https://github.com/lamdu/lamdu"), repoForks = Just 55, repoHomepage = Just "", repoFork = Just False, repoGitUrl = Just (URL "git://github.com/lamdu/lamdu.git"), repoPrivate = False, repoArchived = False, repoCloneUrl = Just (URL "https://github.com/lamdu/lamdu.git"), repoSize = Just 22529, repoUpdatedAt = Just 2018-09-16 19:10:11 UTC, repoWatchers = Just 1481, repoOwner = SimpleOwner {simpleOwnerId = Id 13840193, simpleOwnerLogin = N "lamdu", simpleOwnerUrl = URL "https://api.github.com/users/lamdu", simpleOwnerAvatarUrl = URL "https://avatars2.githubusercontent.com/u/13840193?v=4", simpleOwnerType = OwnerOrganization}, repoName = N "lamdu", repoLanguage = Just (Language "Haskell"), repoDefaultBranch = Just "master", repoPushedAt = Just 2018-09-06 18:54:47 UTC, repoId = Id 2616288, repoUrl = URL "https://api.github.com/repos/lamdu/lamdu", repoOpenIssues = Just 6, repoHasWiki = Just True, repoHasIssues = Just True, repoHasDownloads = Just True, repoParent = Nothing, repoSource = Nothing, repoHooksUrl = URL "https://api.github.com/repos/lamdu/lamdu/hooks", repoStargazersCount = 1481}
```
```
*Search> fetchHelpWanted
Issue {issueClosedAt = Just 2015-07-17 16:24:00 UTC, issueUpdatedAt = 2015-07-17 16:24:00 UTC, issueEventsUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/issues/63/events", issueHtmlUrl = Just (URL "https://github.com/ComputerScienceHouse/csh-eval/issues/63"), issueClosedBy = Nothing, issueLabels = [IssueLabel {labelColor = "207de5", labelUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/labels/Improvement", labelName = N "Improvement"},IssueLabel {labelColor = "d4c5f9", labelUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/labels/help-wanted", labelName = N "help-wanted"}], issueNumber = 63, issueAssignees = [SimpleUser {simpleUserId = Id 2439413, simpleUserLogin = N "dgonyeo", simpleUserAvatarUrl = URL "https://avatars0.githubusercontent.com/u/2439413?v=4", simpleUserUrl = URL "https://api.github.com/users/dgonyeo"}], issueUser = SimpleUser {simpleUserId = Id 1028704, simpleUserLogin = N "gambogi", simpleUserAvatarUrl = URL "https://avatars2.githubusercontent.com/u/1028704?v=4", simpleUserUrl = URL "https://api.github.com/users/gambogi"}, issueTitle = "Establish Nightly Builds", issuePullRequest = Nothing, issueUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/issues/63", issueCreatedAt = 2015-06-18 18:01:54 UTC, issueBody = Just "We need to ensure that a fresh build will always work. Recently we ran into an issue where one of our dependencies updated an the entire build broke. This is not a good sign, and should really be addressed.\n\nThere are two options: Run nightlies to detect these problems, or use stackage to guarantee these problems won't arise.\n\nOne or both are acceptable.\nThis should be a good resource to get started: https://circleci.com/docs/nightly-builds\n", issueState = StateClosed, issueId = Id 89357950, issueComments = 2, issueMilestone = Nothing},Issue {issueClosedAt = Nothing, issueUpdatedAt = 2015-05-04 17:20:30 UTC, issueEventsUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/issues/35/events", issueHtmlUrl = Just (URL "https://github.com/ComputerScienceHouse/csh-eval/issues/35"), issueClosedBy = Nothing, issueLabels = [IssueLabel {labelColor = "91ca55", labelUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/labels/Additions", labelName = N "Additions"},IssueLabel {labelColor = "f7c6c7", labelUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/labels/UI", labelName = N "UI"},IssueLabel {labelColor = "d4c5f9", labelUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/labels/help-wanted", labelName = N "help-wanted"}], issueNumber = 35, issueAssignees = [], issueUser = SimpleUser {simpleUserId = Id 1028704, simpleUserLogin = N "gambogi", simpleUserAvatarUrl = URL "https://avatars2.githubusercontent.com/u/1028704?v=4", simpleUserUrl = URL "https://api.github.com/users/gambogi"}, issueTitle = "Generic Evaluation Voting Interface", issuePullRequest = Nothing, issueUrl = URL "https://api.github.com/repos/ComputerScienceHouse/csh-eval/issues/35", issueCreatedAt = 2015-05-04 17:18:51 UTC, issueBody = Just "As @harlanhaskins brought up in #34, it might be nice to have a generic online way to collect votes. \n\nI'll edit this with more details later\n", issueState = StateOpen, issueId = Id 73075242, issueComments = 0, issueMilestone = Nothing}
```

File: issue-wanted.cabal [hunk 1]
```diff
@@ -75,7 +78,6 @@ test-suite issue-wanted-test
 
   build-depends:       base-noprelude
                      , relude
-                     , issue-wanted
```
Comment (ChShersh) [inline issue-wanted.cabal]: You don't need to remove `issue-wanted` dependency here

File: test/Spec.hs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import Prelude
```
Comment (ChShersh) [inline test/Spec.hs]: I think this import is not required here

**Comment by ChShersh** [comment]
Good start! We definitely can work with this.

**Pull request merged**